bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -198,6 +199,9 @@ private synchronized static void initializeResourceLimiter(BigtableOptions optio
    */
   private BigtableClusterName clusterName;
 
+  private ThrottlingClientInterceptor throttlingClientInterceptor =
+      new ThrottlingClientInterceptor();
+
```

Comment:
This doesn't appear to be used in the channel creation. Also would be possible to construct this in the constructor, passing it the resource limiter and avoiding the enable() bit?